app/src/main/java/com/microsoft/office365/snippetapp/Snippets/EmailSnippets.java

```diff
@@ -257,6 +257,38 @@ public Boolean addItemAttachment(
     }
 
     /**
+     * Deletes all attachments from a message attachment collection and update
+     * the message in the Drafts folder
+     *
+     * @param mailId  The id of the message to update
+     * @return boolean. Success flag. True if attachments are deleted
+     */
+    public boolean removeEmailAttachments(String mailId) throws ExecutionException, InterruptedException {
+        if (mailId.length() == 0)
```

Comment:
Can remove this check and just let the method throw an error if the mailId is truly blank.